install/kubernetes/cilium/values.schema.json

```diff
@@ -1603,6 +1603,12 @@
     },
     "egressGateway": {
       "properties": {
+        "NATMAPStatsEntries": {
+          "type": "integer"
+        },
+        "NATMAPStatsInterval": {
+          "type": "string"
+        },
         "enabled": {
           "type": "boolean"
         },
```

Comment:
I'm unsure about explicitly grouping these with `egressGateway`. That's the main use-case atm, but it's really more of a generic feature - and useful beyond just EGW. For instance I expect it would have also helped users to trace down scalability issues like https://github.com/cilium/cilium/pull/31082.

Reply:
@julianwiedmann Good point, I nested it under a new key: `nat`.